src/styles/style.css

```diff
@@ -0,0 +1,275 @@
+body {
```

Comment:
Ideally, you don't want to keep all CSS declarations in the single file. This project is relatively small and your CSS file is already huge.

There are two ways to make it cleaner.

1. CSS modules: https://github.com/css-modules/css-modules which follows the pattern of styling single component per css file.

2. Inline styling in the React component: https://reactjs.org/docs/dom-elements.html#style

First option might be an overhead for such a small project but inline styles is a good solution in this case.

Reply:
Will do the first option. Have been already brainwashed, by previous "TSEs" about inline styling 😄 .